style.css

```diff
@@ -1,169 +1,112 @@
-:root {
+* {
   border: 0;
   margin: 0;
   padding: 0;
+}
+
+:root {
+  box-sizing: border-box;
 
   --hover-underline-color: aqua;
   --color: #1f3b58;
-  --hover-color: #051e38;
-  --font-family: 'Poppins' georgia, 'Times New Roman', times, serif;
+  --hover-color: #374711;
+  --font-family: roboto, sans-serif;
   --webkit-background-clip: text;
+  --moz-background-color: rgb(9, 2, 49);
 }
 
-/* Header */
-.sub-container {
+.main-container {
+  padding-left: 24px;
+  padding-right: 24px;
   display: flex;
-  justify-content: space-between;
-  text-align: center;
-  margin-top: 10px;
-  align-items: center;
 }
 
-.logo-cont {
-  width: 120px;
+nav {
+  width: 98%;
+  height: 60px;
+  background: var(--moz-background-color);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}
 
-  /* height: 30px; */
-  text-decoration: none;
+.menu {
+  padding-right: 50px;
 }
 
-.header-font {
-  width: 60px;
-  height: 30px;
+.logo {
+  font-weight: 700;
+  font-family: var(--font-family);
   color: var(--color);
-  border: 2px;
-  visibility: hidden;
+  display: flex;
+  font-size: 18px;
+  line-height: 24px;
 }
 
-.logo-cont:hover,
-.font:hover {
-  text-decoration: underline;
-  text-decoration-color: var(--hover-underline-color);
-  color: var(--hover-color);
+.main-header {
+  background-size: 256px;
+  height: 100vh;
+  width: 100%;
+  background-color: rgb(25, 12, 99);
+  display: flex;
+  background-repeat: no-repeat;
+  background-position: bottom right -110px;
+  background-image: url("./images/header-shapes\ mobile.png");
 }
 
-/* End of Header */
-
-/* Main Section */
-
-.main-container {
-  padding: 10px;
-
-  /* position: absolute; */
+.main-header-content {
```

Comment:
- [ ] Please make sure that you are using the same background image as in [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238)

![image](https://user-images.githubusercontent.com/74744787/158868532-da0749bd-ae9c-46e9-95fe-6b096b43d4e5.png)